java-security/src/main/java/com/sap/cloud/security/token/SapIdToken.java

```diff
@@ -39,6 +39,16 @@ public Service getService() {
 		return Service.IAS;
 	}
 
+	/**
+	 * Gets the token issuer domain that is required to check trust in the issuing identity service.
+	 * If tenants use a custom domain for the identity service, claim {@link SapIdToken#IAS_ISSUER} needs to be
+	 * used for trust checks instead of claim {@link TokenClaims#ISSUER}. It contains the internal domain of the identity
+	 * service. External customer domains are not trusted.
+	 * <br><br>
+	 * Use {@link SapIdToken#getCustomIssuer()} instead to get the custom domain if one is used.
```

Comment:
I chose to remove it because its conditional logic checked for existence of IAS_ISSUER and only if available, would return ISSUER. I found this behaviour a bit strange but do not want to cause issues by making a variant of it public with a different logic.